bot.py

```diff
@@ -28,25 +29,24 @@
 
 startTime = 0
 
-bot = discord.Client(intents=intents)
+bot = commands.Bot(command_prefix="/", intents=intents)
```

Comment:
Why are you using commands.Bot instead of discord.Client? Using that would mean that people could type `/misc ping` into the chat (or ! if the command_prefix was ! ), which is not the functionality of slash commands?